download-maven-plugin/src/it/basicGet/verify.groovy

```diff
@@ -1,5 +1,5 @@
 import junit.framework.Assert;
 
 File f = new File(basedir, "target/lolcatsdotcomcm90ebvhwphtzqvf.jpg");
-Assert.assertTrue("File " + f.absolutePath + " does not exist", f.exists());
+Assert.assertTrue("File " + f.absolutePath + " does exist", f.exists());
```

Comment:
Actually, it is not: the message is the error shown to developer in case the assertion fails. In that case, if `f.exists()` fail, we want the developer to see as error message that the file does *not* exisot